packages/@okta/vuepress-site/docs/guides/customize-authz-server/create-rules-for-policy/index.md

```diff
@@ -32,10 +16,33 @@ Then the access token that is granted has a lifetime of, for example, one hour.
 
 You can also use rules to restrict grant types, users, or scopes. For example, you could prevent the use of all scopes other than `openid` and `offline_access` by only creating rules that specifically mention those two scopes. This means you would have to:
 
-1. Not create any rules that match "any scopes", and
-2. Ensure that all of your rules only match the `openid` and/or `offline_access` scopes.
+* Not create any rules that match "any scopes", and
+
+* Ensure that all of your rules only match the `openid` and/or `offline_access` scopes.
 
-Any request that is sent with a different scope won't match any rules and will consequently fail.
+Any request that is sent with a different scope won't match any rules and consequently fails.
+
+To create a rule for a policy:
+
+1. In the Developer Console, navigate to **API > Authorization Servers**.
+2. Select the name of an Authorization Server, and then select **Access Policies**.
+3. Select the name of an access policy, and then select **Add Rule**.
+4. Enter a **Name** for the rule, and then use the following fields to define the rule logic:
+    * **IF Grant type is:** Select one or more OAuth 2.0 grant types. See [Choosing an OAuth 2.0 flow](/docs/concepts/oauth-openid/#choosing-an-oauth-2-0-flow) for more information on understanding the type of OAuth flow (grant type) that you should use.
+    * **AND User is:** Select one of the following:<br>
+        **Any user assigned the app:** The rule applies to any user that is assigned to the app.<br>
+        **Assigned the app and a member of one of the following:** The rule applies to users that are assigned to the app and a member of one or more groups that you specify. You can also specify a list of users to apply the rule to.<br>
+    > **Note:** The app must be assigned to this rule's policy.
```

Comment:
I think this needs to be indented another two spaces to go under the bullet point where it applies.